src/routes/auth.ts

```diff
@@ -69,7 +69,7 @@ router.post(
       } else {
         // Send signinKey key (ONLY IN DEV)
         logger.warn(
-          `--DEV ONLY-- Email not sent to ${signinKey.email}. SignIn key sent in response!`
```

Comment:
So I actually modified the logger so that the environment is prepended to all messages. With that setup it would be unnecessary to clarify that a message is development only as it simply won't appear in any other context.